pkg/providers/instancetype/suite_test.go

```diff
@@ -1135,6 +1136,42 @@ var _ = Describe("Instance Types", func() {
 			}
 		})
 		It("shouldn't report more resources than are actually available on instances", func() {
+			awsEnv.EC2API.DescribeSubnetsOutput.Set(&ec2.DescribeSubnetsOutput{
```

Comment:
Why did you need to add this?

Reply:
Since we now reject and remove an instance types that have 0 offerings, we need to make sure the describe subnets returns a zone so that we can get an instance type that has at least one offering for the test below.